README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# hi! welcome to my repo. 
+## my name is sanaa imami and i'm a rising senior at the lab schools. 
+i intend to double major in **psychology** and **middle eastern studies**. 
+i’ve taken a rudimentary cs course before. here are some topics we spent time learning about:
+* [html](https://en.wikipedia.org/wiki/HTML)
+* [java](https://en.wikipedia.org/wiki/Java_(software_platform))
+* [CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+* [SVG](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics)
+
+i remember a bit of these topics, but not enough to apply in a practical setting. 
+i’m excited to expand upon my very limited CS knowledge in the next few weeks!
```

Comment:
This was a great introduction to a little bit about yourself, and I really like how you referred back to your past experience with programming languages. I can't find any pressing flaws about your code but one thing that Prof. Soltoff recommended to me was to leave blank lines between markdown elements to avoid problems in future code. Otherwise, this was fantastic and you did a great job using a variety of markdown elements

Reply:
@Sophiaegrari That is a good point to bring up!